portfolio_sim/portfolio_simulator.py

```diff
@@ -47,16 +47,22 @@ def simulate_strategy(self, strategy):
             action_hist[date] = portfolio_order
             for company in portfolio_order:
                 if portfolio_order[company] > 0:
-                    if money >= daily_data[company].Close * portfolio_order[company]:
+                    if money >= daily_data[company].Close * portfolio_order[
+                            company]:
                         portfolio[company] += portfolio_order[company]
-                        money -= daily_data[company].Close * portfolio_order[company]
+                        money -= daily_data[company].Close * portfolio_order[
+                            company]
```

Comment:
Is this the current black formatting recommendation?

Reply:
Personally think this formatting is less readable, do you think we should change the formatting rules?